go/server.go

```diff
@@ -243,15 +256,29 @@ func main() {
 	}
 
 	r := gin.Default()
-	r.LoadHTMLFiles("templates/index.tmpl")
+	r.LoadHTMLFiles("templates/index.tmpl", "templates/oauth-response.tmpl")
 	r.Static("/static", "./static")
 
 	r.GET("/", func(c *gin.Context) {
 		c.HTML(http.StatusOK, "index.tmpl", gin.H{
+			"plaid_environment":        "sandbox", // Switch this environment
```

Comment:
For consistency with our other quickstarts, I suggest setting this to an environment variable such as `PLAID_ENV`.

Reply:
Thanks, let's tackle this in a separate PR since the golang quickstart is currently hardcoded in a couple of places for sandbox.

Reply:
Let's tackle this in a separate PR since the golang quickstart is currently hardcoded to sandbox in a couple of places including https://github.com/plaid/quickstart/blob/master/go/server.go#L39 which is harder to change.